LastWork(newWork)/Views/index/index.js

```diff
@@ -42,3 +44,58 @@ const checkUserDatas = function (user, pass) {
     }
     return false;
 };
+
+function check(elem) {
+    return elem.checked === true;
+}
+
+const conslog = function (event) {
+    event.stopPropagation();
+    event.preventDefault();
+
+    const CandidateName = document.querySelector('#CandName').value;
+    const InterviewerName = document.querySelector('#IntervName').value;
+    const IntervDate = document.querySelector('#dateForm').value;
+
+    const TechnicalLevel = document.querySelectorAll('#b');
```

Comment:
If the id is unique you will always have one element that this query will return; as such you don't need the All. Unless the id is not unique which is not an option.